manifests/generated/operator-csv.yaml.in

```diff
@@ -499,6 +499,21 @@ spec:
           - update
           - delete
           - patch
+        - apiGroups:
+          - ""
+          resources:
+          - persistentvolumes
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - secrets
+          verbs:
+          - create
+          - delete
```

Comment:
if the hostpath is truly sensitive information, maybe we can store this secret data in a single namespace, like perhaps the kubevirt install namespace. Then we wouldn't need permissions to globallly view/delete secrets

Reply:
I wish I could, but I am mounting the secret in the attachment pod so I can then read it with virt-handler. This also means that anyone able to exec into the attachment pod can read the path, but that is a problem with any secret mounted in a pod. But because it is mounted in the attachment pod, it needs to exist in the namespace the pod is running in.